Command/InstallZipCodeDataCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nbush
+ * Date: 4/4/18
+ * Time: 1:49 PM.
+ */
+
+namespace MauticPlugin\MauticEnhancerBundle\Command;
+
+use Mautic\CoreBundle\Command\ModeratedCommand;
+use MauticPlugin\MauticEnhancerBundle\Entity\PluginsEnhancerCityStateZip;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class InstallZipCodeDataCommand extends ModeratedCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('mautic:integration:enhancer:zipcode-import')
+            ->setDescription('Imports allCountries.txt zipcode, city, state (US Primary only)')
+            ->setHelp('final documentation for file state and location will be addressed here (i.e., downloaded, zipped, etc')
+            ->addArgument('filename', InputArgument::REQUIRED, 'name of file to import');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $em       = $this->getContainer()->get('doctrine.orm.entity_manager');
+        $filename = $input->getArgument('filename');
+        if (!file_exists($filename)) {
+            // determine required stream interfaces?
```

Comment:
Needs to throw an error, or something. Same trap for (!$file).

Reply:
Let's make this method also do the download and unzip if the file doesn't exist.